packages/ra-keycloak/src/LoginPage.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { useLogin } from 'ra-core';
```

Comment:
```suggestion
import { useLogin } from 'react-admin';
```

Reply:
As explained, I instead made sure we always import from `ra-core`